react/index.js

```diff
@@ -43,9 +43,9 @@ class Header extends Component {
   }
 
   render() {
-    const { logoUrl, logoTitle, showSearchBar, showLogin, linkUrl } = this.props
-
+    const { logoUrl, logoTitle, showSearchBar, showLogin, linkUrl, runtime: {hints} } = this.props
```

Comment:
fix hints destructuring:

```suggestion
    const { logoUrl, logoTitle, showSearchBar, showLogin, linkUrl, runtime: { hints: mobile } } = this.props
```

Reply:
Fixed